mono/utils/mono-threads-mach.c

```diff
@@ -33,7 +33,7 @@ mono_threads_suspend_init (void)
 	mono_threads_init_dead_letter ();
 }
 
-#if defined(HOST_WATCHOS) || defined(HOST_TVOS)
+#if defined(HOST_WATCHOS)
```

Comment:
Note, the guarded code by `HOST_TVOS` wasn't really guarded _before_ as `HOST_TVOS` didn't exist. async suspend and friends are available on tvOS, so this was wrong.